modules/references/client/components/Navigation.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Navigation is a react component.
+ * It can contain three different buttons: Back, Next, Submit. Each of them has a related property onBack, onNext, onSubmit
+ *
+ */
+export default function Navigation(props) {
+  const back = (
+    <button
+      type="button"
+      className="btn btn-action btn-link"
+      aria-label="Previous section"
+      onClick={props.onBack}>
+      <span className="icon-left"></span>
+      Back
+    </button>
+  );
+
+  const next = (
+    <button
+      type="button"
+      className="btn btn-action btn-primary"
+      aria-label="Next section"
+      onClick={props.onNext}
+      disabled={props.tabDone < props.tab}>
+      Next
+    </button>
+  );
+
+  const submit = (
+    <button
+      className="btn btn-action btn-primary"
+      aria-label="Submit reference"
+      onClick={props.onSubmit}
+      disabled={props.tabDone < props.tabs - 1 || props.disabled}>
+      Submit
+    </button>
+  );
+
+  return (
+    <div className="text-center">
+      {(props.tab > 0) ? back : null}
```

Comment:
This shorter form should work:
```js
{props.tab > 0 && back}
```

For clarity, let's call these components "backButton".

The same in other places in this PR where ternary + `null` is used.

Reply:
:ok_hand: I thought this syntax was unclear, but @agatatalita already explained it to me.

Reply:
I just love the simplicity of:

```
{ isEnabled && <Component> }
```
vs
```
{ isEnabled ? <Component> : null }
```

🙃

Reply:
done